reconciler.go

```diff
@@ -82,6 +86,7 @@ func (r *Reconciler[T, PT]) Reconcile(ctx gocontext.Context, req ctrl.Request) (
 		return ctrl.Result{Requeue: true, RequeueAfter: 2 * time.Minute}, err
 	}
 
+	// TODO: What does this do ? how does it set status if it is recieved in reconcile
```

Comment:
@moshloop @adityathebe 
What does this do ? I don't think this flow is ever invoked, since the client code updates the status field

Reply:
https://github.com/flanksource/mission-control/commit/59440ab0655debaa7220935fd71314e51cb791b4